### PR TITLE
add ablang

### PR DESCRIPTION
**Pull request by mbestipa**
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mbestipa** [comment]
I confirm I'm willing to be listed here as a maintainer

**Comment by mbestipa** [comment]
@conda-forge/help-python, ready for review

File: recipes/ablang2/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channel_sources:
```
Comment (xhochy) [inline recipes/ablang2/conda_build_config.yaml:1]: Please remove this file. Packages on conda-forge should only depend on conda-forge packages.

File: recipes/ablang2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 7f01bec5c5a5d5c270fa078cdd923d3a98bdc2b93538b17d46231db2d783676d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >3.6
```
Comment (xhochy) [inline recipes/ablang2/meta.yaml:19]:
```suggestion
    - python >=3.6
```
Or should it be 3.7?

File: recipes/ablang2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - setuptools
+  run:
+    - python >3.6
```
Comment (xhochy) [inline recipes/ablang2/meta.yaml:23]:
```suggestion
    - python >=3.6
```

File: recipes/ablang2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - pytorch >1.9
```
Comment (xhochy) [inline recipes/ablang2/meta.yaml:24]: Same

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by mbestipa** [comment]
@xhochy I've changed things as the windows build fails without using other channels (no pytorch v2), I'm fine with things not being available in windows

**Comment by mbestipa** [comment]
@conda-forge/help-python can this be reviewed again?

File: recipes/ablang2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ablang2-{{ version }}.tar.gz
+  sha256: 7f01bec5c5a5d5c270fa078cdd923d3a98bdc2b93538b17d46231db2d783676d
+
+build:
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/ablang2/meta.yaml:13]: It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

**Comment by mbestipa** [comment]
@ocefpaf is this to your liking?

**Comment by ocefpaf** [comment]
> @ocefpaf is this to your liking?

My liking doesn't matter here 😄 . But we do need to have some recipe consistency for out build system.

**Comment by mbestipa** [comment]
> > @ocefpaf is this to your liking?
> 
> My liking doesn't matter here 😄 . But we do need to have some recipe consistency for out build system.

of course, thanks for reviewing it.

**Pull request merged**